predicate/predicate.py

```diff
@@ -174,4 +164,85 @@ def _regex(self, lookup_model, lookup_field):
     def _iregex(self, lookup_model, lookup_field):
         return bool(re.search(self.value, lookup_field, flags=re.I))
 
+    def _in(self, instance, lookup_value):
+        """
+        Evaluates whether the lookup list and this value list intersect.
+        """
+        return bool(set(lookup_value) & set(self.value))
+
+    def _any(self, instance, lookup_value):
+        """
+        Evaluates whether the lookup list contains this value.
+        """
+        return self.value in lookup_value
```

Comment:
Great catch. Maybe we can replicate the `_any` functionality for dictionaries then by overriding `_exact` and special-casing for list types? Then drop the `_any` altogether?